solutions/swb-ui/scripts/updatePlugins.sh

```diff
@@ -11,7 +11,7 @@ for plugin in *; do #Iterate through swb-plugins
         pluginUpper=$(echo "${plugin}" | tr “[a-z]” “[A-Z]”);
         if ! $(jq --arg pluginName $pluginPackageName 'any(.projects[].packageName == $pluginName; .)' ../../rush.json); then
             echo "Adding $pluginPackageName to rush.json";
-            jq --arg pluginName $pluginPackageName --arg pluginFolder "solutions/swb-plugins/$plugin" '.projects += [{"packageName": $pluginName, "projectFolder": $pluginFolder, "reviewCategory": "production", "shouldPublish": true}]' ../../rush.json > ../../output_rush.json;
+            jq --arg pluginName $pluginPackageName --arg pluginFolder "solutions/swb-plugins/$plugin" '.projects += [{"packageName": $pluginName, "projectFolder": $pluginFolder, "reviewCategory": "production", "shouldPublish": false}]' ../../rush.json > ../../output_rush.json;
```

Comment:
Why set shouldPublish equal to `false`?

Reply:
setting it to true will set our pipeline to publish the package as an `npm` package to NPM

Reply:
understood, then we'll keep it to `false`, however for it to build everything we'll need to tell users to set it to `true`.

Reply:
hmm, I was able to test it from this branch with the value set to `false` and everything work as expected. Can you test on your side to see if everything build correctly for you as well?

Reply:
It build correctly with it set to `false` so we'll keep it like that. No need to set to `true`.